src/components/formik-forms/formik-radio-button.jsx

```diff
@@ -10,24 +10,26 @@ require('./formik-radio-button.scss');
 require('../forms/row.scss');
 
 const FormikRadioButtonSubComponent = ({
-    buttonValue,
```

Comment:
why did you change the name of buttonValue?  Also, I'm confused again, what's the difference between field.value and value?

Reply:
Yeah, this is one of the most confusing things about formik's handling of radio buttons...

So, when you give the formik `<Field>` component a `subComponent` prop (which you need to do for radio buttons because there's no built-in handling of them, unlike inputs and selects... which is annoying), it automatically passes it a `field` prop. And what's weird is, `field.value` is the value of the _radio button group_, not this individual button. I can't figure out why this is, but that's the `field.value` when I inspect/log it.

Meanwhile, `value` is the value of this individual button. 

So to see if this button should be currently "selected", you need to compare those two kinds of value.

Reply:
oh, and I thought that `value` would be simpler than `buttonValue`. What do you think?

Reply:
I see (maybe :) ). In this example, are they using a strategic value of id to get around this? https://codesandbox.io/s/pjqp3xxq7q

Reply:
In that example, I think this section of `CheckboxGroup` is doing a lot of work:

```
            return React.cloneElement(child, {
              field: {
                value: value.includes(child.props.id),
                onChange: this.handleChange,
                onBlur: this.handleBlur
              }
            });
```

Reply:
Sorry, I meant the RadioGroup. In the UI (and I think the code too) the CheckboxGroup is a separate thing.  Sorry to to get stuck on this point. Just want to make sure I have a better understanding of how it works.

Reply:
Sorry, yes, your previous comment is exactly right. In that codesandbox example, they use the `id` prop to set `value`, and they use `field.value` to compare to `id` to determine if the button is selected.

Make sense? I guess formik is acting like all the radio buttons are a single "field", or something? I don't totally understand it, to tell the truth.